Traducir.Web.Net/wwwroot/lib/bootstrap/LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```

Comment:
can we use bootstrap from a CDN instead of including it on the project?

Reply:
This was (you guessed it) automatically added by Visual Studio. But if you look at `_Layout.cshtml` you'll see that the local files are used only in the development environment, in production a CDN is indeed used.

Reply:
👍